modules/apps/29-fee/types/errors.go

```diff
@@ -15,4 +15,5 @@ var (
 	ErrForwardRelayerAddressNotFound = sdkerrors.Register(ModuleName, 8, "forward relayer address not found")
 	ErrFeeNotEnabled                 = sdkerrors.Register(ModuleName, 9, "fee module is not enabled for this channel. If this error occurs after channel setup, fee module may not be enabled")
 	ErrRelayerNotFoundForAsyncAck    = sdkerrors.Register(ModuleName, 10, "relayer address must be stored for async WriteAcknowledgement")
+	ErrFeeModuleLocked               = sdkerrors.Register(ModuleName, 11, "the fee module is currently locked")
```

Comment:
Nit:

It might be worth making this error message more descriptive so that people know what/why the module is "locked". wdyt? 

Maybe something added to the end like 

```
The fee module is locked if a severe bug is present
```